.github/ISSUE_TEMPLATE/design-discussion.md

```diff
@@ -0,0 +1,30 @@
+---
+name: Design discussion
```

Comment:
Not sure why, but you've got a folder `github/` rather than the hidden `.github/`

Reply:
Huh, at least it looked like that in the review window. Still not sure why you're getting this in your PR.